librz/include/rz_core.h

```diff
@@ -989,13 +989,13 @@ RZ_API bool rz_core_diff_show_function(RzCore *core, RzCore *core2, ut64 addr, b
 RZ_API void rz_core_clippy(RzCore *core, const char *msg);
 
 /* watchers */
-RZ_API void rz_core_cmpwatch_free(RzCoreCmpWatcher *w);
-RZ_API RzCoreCmpWatcher *rz_core_cmpwatch_get(RzCore *core, ut64 addr);
-RZ_API int rz_core_cmpwatch_add(RzCore *core, ut64 addr, int size, const char *cmd);
-RZ_API int rz_core_cmpwatch_del(RzCore *core, ut64 addr);
-RZ_API int rz_core_cmpwatch_update(RzCore *core, ut64 addr);
-RZ_API int rz_core_cmpwatch_show(RzCore *core, ut64 addr, int mode);
-RZ_API int rz_core_cmpwatch_revert(RzCore *core, ut64 addr);
+RZ_API void rz_core_cmpwatch_free(RZ_NONNULL RzCoreCmpWatcher *w);
+RZ_API RzCoreCmpWatcher *rz_core_cmpwatch_get(RZ_NONNULL RzCore *core, ut64 addr);
+RZ_API bool rz_core_cmpwatch_add(RZ_NONNULL RzCore *core, ut64 addr, int size, const char *cmd);
+RZ_API bool rz_core_cmpwatch_del(RZ_NONNULL RzCore *core, ut64 addr);
+RZ_API bool rz_core_cmpwatch_update(RZ_NONNULL RzCore *core, ut64 addr);
+RZ_API void rz_core_cmpwatch_show(RZ_NONNULL RzCore *core, ut64 addr, RzOutputMode mode);
+RZ_API bool rz_core_cmpwatch_revert(RZ_NONNULL RzCore *core, ut64 addr);
```

Comment:
do we want to move also these to rz_cmp.h?

Reply:
Moving them to `rz_cmp.h` is problematic since `core.c` uses the function `rz_core_cmpwatch_free`.

Reply:
yes, absolutely. rz_cmp is just a part of rzcore AFAIK.